mappings/net/minecraft/world/gen/random/BaseSimpleRandom.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_6566 net/minecraft/world/gen/random/BaseSimpleRandom
+	FIELD field_34582 FLOAT_EPSILON F
```

Comment:
```suggestion
	FIELD field_34582 FLOAT_MULTIPLIER F
```
The term `epsilon` is usually used for equality check, which is not the case here.